pkg/networkfs/resolvconf/resolvconf.go

```diff
@@ -65,8 +65,8 @@ func GetLastModified() ([]byte, string) {
 	return lastModified.contents, lastModified.sha256
 }
 
-// RemoveReplaceLocalDns looks for localhost (127.*) entries in the provided
-// resolv.conf, removing local nameserver entries, and, if the resulting
+// RemoveReplaceLocalDns looks for localhost (127/8 and ::1/128) entries in the
```

Comment:
Probably.  I just decided to be syntactically incorrect in the same way the original comment was.